lib/getAssetInfo.ts

```diff
@@ -0,0 +1,42 @@
+import prisma from "@/lib/prisma";
+import { env } from "process";
+
+export async function getAssetPath(id: string) {
+  const directusFileItem = await prisma.directusFiles.findFirst({
+    where: { id },
+  });
+  if (directusFileItem === null) throw new Error("No directusFilesItem found");
+  return (
+    env.DIRECTUS_URL +
+    "assets/" +
+    directusFileItem.id +
+    "/" +
+    directusFileItem.filenameDownload
+  );
+}
+
+export interface GetImageInfo {
+  path: string;
+  id: string;
+  filename: string;
+}
+
+export async function getImageInfo(id: string) {
+  if (id === "") throw new Error("No id found");
```

Comment:
Why are you checking if the string is empty?

Reply:
Because it the image can be empty if we decide not to add both images and it gets called for both images even if they are empty.

Reply:
Then don’t call the function if the image is null.